config/custom_items/README.md

```diff
@@ -1 +1,75 @@
-For details on how to use the custom item system, please refer to [the custom item repository](https://github.com/Baystation12/custom-items).
\ No newline at end of file
+## How to add a custom item
+1. Pick the most relevant directory for your item, and examine the template inside.
+
+2. Create a .json file in the directory with the following naming scheme: ckey-itemtype.json (e.g. babydoll-necklace.json)
+
+3. Fill out the provided template in the directory of your choice. See chart below for an explanation of the values.
+
+| Key                    | Expected Value   | Function                                                                                                                |
+|------------------------|------------------|-------------------------------------------------------------------------------------------------------------------------|
+| ckey                   | string           | Your ckey. This is not quite the same as your BYOND key, ask an admin or check the BYOND docs if you are unsure.        |
+| character_name         | string           | The name of the character the item should spawn with.                                                                   |
+| item_name              | string           | The name of your custom item ingame. For kits, the name of the kit product.                                             |
+| item_desc              | string           | The description of your custom item ingame. For kits, the descriptor for the kit product.                               |
+| item_icon              | file path        | The location of the icon file to load for your item (ex. icons/custom_icon_file.dmi)                                    |
```

Comment:
May be worth having a standardized relative path for this (possibly as a config option/define?). This is always used by people with marginal understanding of project structure, and even I find navigating the file tree confusing when you have to include multiple `..`, etc.

Reply:
I could make it use a config value but if anything that's adding a step to the confusion as the server could set it to anything.

Reply:
Done, commented out by default.